spec/timezone.html

```diff
@@ -619,12 +619,12 @@ <h1>BuiltinTimeZoneGetInstantFor ( _timeZone_, _dateTime_, _disambiguation_ )</h
         1. Let _offsetAfter_ be ? GetOffsetNanosecondsFor(_timeZone_, _dayAfter_).
         1. Let _nanoseconds_ be _offsetAfter_ − _offsetBefore_.
         1. If _disambiguation_ is *"earlier"*, then
-          1. Let _earlier_ be ? AddDateTime(_dateTime_.[[ISOYear]], _dateTime_.[[ISOMonth]], _dateTime_.[[ISODay]], _dateTime_.[[ISOHour]], _dateTime_.[[ISOMinute]], _dateTime_.[[ISOSecond]], _dateTime_.[[ISOMillisecond]], _dateTime_.[[ISOMicrosecond]], _dateTime_.[[ISONanosecond]], 0, 0, 0, 0, 0, 0, 0, 0, 0, −_nanoseconds_, *"constrain"*).
+          1. Let _earlier_ be ? AddDateTime(_dateTime_.[[ISOYear]], _dateTime_.[[ISOMonth]], _dateTime_.[[ISODay]], _dateTime_.[[ISOHour]], _dateTime_.[[ISOMinute]], _dateTime_.[[ISOSecond]], _dateTime_.[[ISOMillisecond]], _dateTime_.[[ISOMicrosecond]], _dateTime_.[[ISONanosecond]], _dateTime_.[[Calendar]], 0, 0, 0, 0, 0, 0, 0, 0, 0, −_nanoseconds_, *undefined*).
```

Comment:
Rather than passing undefined through to CalendarDateAdd, I think for consistency we should do something like 

> 1. Let _addOptions_ be ! OrdinaryObjectCreate(*null*).
> 1. Let _earlier_ be ? AddDateTime(..., _addOptions_).

(search for _addOptions_ in the spec text to see how it's done elsewhere)

Reply:
> Rather than passing undefined through to CalendarDateAdd, I think for consistency we should do something like
> 
> > 1. Let _addOptions_ be ! OrdinaryObjectCreate(_null_).
> > 2. Let _earlier_ be ? AddDateTime(..., _addOptions_).
> 
> (search for _addOptions_ in the spec text to see how it's done elsewhere)

this is not observable right? why pick a less optimal way to express the same unobservable behavior  in the spec?

Reply:
I believe it would be observable in a case like the following:
```js
class C extends Temporal.Calendar {
    constructor() { super('iso8601'); }
    dateAdd(date, duration, options) {
        console.log(options);
        return super.dateAdd(date, duration, options);
    }
}
plain = Temporal.PlainDateTime.from('2021-03-14T02:30').withCalendar(new C());
plain.toZonedDateTime('America/Vancouver');
```
This would log `undefined` or `[object Object]` depending on what we passed through to CalendarDateAdd.

Reply:
oic. ok, changed. PTAL